waspc/examples/todoApp/src/client/pages/Task.tsx

```diff
@@ -33,7 +33,7 @@ const Todo = (props: any) => {
   })
 
   if (!task) return <div>Task with id {taskId} does not exist.</div>
-  if (error) return <div>Error occurred! {error}</div>
+  if (error) return <div>Error occurred! {error as any}</div>
```

Comment:
This came with React 18 change, since `error` is `unknown` TS throws the following error
```
This JSX tag's 'children' prop expects a single child of type 'ReactNode', but multiple children were provided.ts(2746)
```

Some info online: https://stackoverflow.com/a/74746893/1502357